api/conf/settings/security.py

```diff
@@ -52,3 +54,12 @@
     USE_X_FORWARDED_HOST = True
     # https://docs.djangoproject.com/en/4.0/ref/settings/#secure-proxy-ssl-header
     SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# Adding DJANGO_SECRET_KEY check
+if (
+    config("DJANGO_SECRET_KEY") == "example_key"
+    and config("ENVIRONMENT", default="local") not in ["local", "development"]
+):
+    raise ImproperlyConfigured(
+        "DJANGO_SECRET_KEY should not be 'example_key' in local or production environment."
+    )
```

Comment:
```suggestion
    )

```
This file should have a blank line at the end